src/test/java/seedu/address/model/ModelManagerTest.java

```diff
@@ -97,11 +98,13 @@ public void getFilteredPersonList_modifyList_throwsUnsupportedOperationException
     public void equals() {
         AddressBook addressBook = new AddressBookBuilder().withPerson(ALICE).withPerson(BENSON).build();
         AddressBook differentAddressBook = new AddressBook();
+        ModBook modBook = new ModBook();
+        ModBook differentModBook = new ModBook();
```

Comment:
`differentModBook` is the same as `modBook`? i know this not v important but a bit strange la, especially since you already created the `ModBookBuilder` util class

Reply:
@bentanjunrong I'll assume you were waiting for the `TypicalModules` class to create the `differentModBook`. I'll update the checks for the `equals()` method now that the typical modules are available, but later you can double check if I missed out anything you intended.